umh.docs.umh.app/content/en/docs/datacontracts/_index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Data Contracts / API"
+menuTitle: "Data Contracts / API"
+description: "This page describes how messages flow in the UMH, which message goes where, how it has to be formatted and how you can create your own structures."
+weight: 1725
+---
+
+## What are Data Contracts
+
+Data Contracts are agreements that define how data is structured, formatted, and
+managed when different parts of a Unified Namespace (UNS) architecture
+communicate. They cover metadata, data models, and service levels to ensure that
+all systems work together smoothly and reliably.
+
+Simply put, data contracts specify where a message is going, the format it must
+follow, how it's delivered, and what happens when it arrives - all based on
+agreed-upon rules and services. It is similar to an API: you send a specific
+message and it triggers a predefined action.
+
+### Example Historian
+
+To give you a simple example, just think about the `_historian` schema. Perhaps
+without realizing it, you have already used the Historian Data Contract by using
+this schema.
+
+Whenever you send a message to a topic that contains the `_historian` schema via
+MQTT, you know that it will be bridged to Kafka and end up in TimescaleDB.
+You could also send it directly into Kafka, and you know that it gets
+bridged to MQTT as well.
+
+But you also know that you have to follow the correct payload and topic
+structure that we as UMH have defined. If there are any issues like a missing
+timestamp in the message, you know that you could look them up in the
+Management Console.
+
```

Comment:
Maybe a screenshot there how to view those information?

Reply:
Seems like, currently there is no misconfigured tag in the demo simulator. During the refactoring of the demo simulator, I would like to add this again and then complement this section with the screenshot